src/nativeMain/kotlin/lumberjack/sawtooth/layout/componentRegistry.kt

```diff
@@ -0,0 +1,16 @@
+package lumberjack.sawtooth.layout
+
+import lumberjack.sawtooth.component.*
+import kotlin.native.concurrent.AtomicReference
+
+internal val _componentRegistry = AtomicReference(mapOf(
+    "msg" to MessageComponent,
```

Comment:
Can expand the aliases. For example, for PatternLayout in Log4J2, message is `m`, `msg`, and `message`. Can apply to others as well.